Packs/ThycoticDSV/Integrations/ThycoticDSV/ThycoticDSV.yml

```diff
@@ -0,0 +1,60 @@
+category: Authentication
+commonfields:
+  id: ThycoticDSV
+  version: -1
+configuration:
+- display: Server URL (e.g. https://example.com)
+  name: url
+  required: true
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Client ID for authenticate
+  hidden: false
+  name: client_id
+  required: true
+  type: 0
+- display: Client Secret for authenticate
+  hidden: false
+  name: client_secret
+  required: true
+  type: 0
+description: Manage credentials for applications, databases, CI/CD tools, and services
+  without causing friction in the development process.
+display: ThycoticDSV
+name: ThycoticDSV
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: Secret name for DSV
+      isArray: false
+      name: name
+      required: true
+      secret: false
+    deprecated: false
+    description: Retrieved secret from DSV
```

Comment:
I still suggest you take a closer look into the phrasing and grammar of descriptions, like this one (but others as well).
Please take a closer look